src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -349,6 +427,47 @@ void MPS::apply_diagonal_matrix(const AER::reg_t &qubits, const cvector_t &vmat)
   apply_matrix(qubits, diag_mat);
 }
 
+void MPS::centralize_qubits(MPS & new_MPS, const reg_t &qubits, 
+			    uint_t &new_first, uint_t &new_last, bool & ordered ) const {
+  new_MPS.initialize(*this);
+  ordered = true;
```

Comment:
We are always overwriting `ordered` argument here, so there's no point on passing it as an argument at all.

Reply:
okay, removed.